README.md

```diff
@@ -704,8 +704,8 @@ The adapter supports python models using [`spark`](https://docs.aws.amazon.com/a
 - `spark_cross_account_catalog` (`default=false`)
   - In Spark, you can query the external account catalog and for that the consumer account has to be configured to
    access the producer catalog.
```

Comment:
I find this sentence extremely unclear. Can anyone help me try to parse/clarify what it means so I can rewrite it?

Reply:
I would say something like this:

> When using Spark Athena workgroup, queries can only be run against catalogs located on the same AWS account. But sometimes you want to make queries to an other catalog located on an external AWS account. This requires an additional parameter on the spark properties setup. Set it to true when you want to query external catalogs.  If this flag is set to true, you can use the syntax `external_catalog_id/database.table` to access the external table on the external catalog (ex: 999999999999/mydatabase.cloudfront_logs` where 999999999999 is the external catalog ID)

And I will remove the next line

Reply:
Fixed in e5ac7db!